app/handlers/newflow/student_signup/signup_form.rb

```diff
@@ -61,11 +64,23 @@ def handle
         end
 
         agree_to_terms
+
+        if options[:is_BRI_book]
```

Comment:
This same block is in the educator signup flow. Can it be in one place and not duplicated?